wire/modules/AdminTheme/AdminThemeUikit/AdminThemeUikit.module

```diff
@@ -578,6 +578,24 @@ class AdminThemeUikit extends AdminThemeFramework implements Module, Configurabl
 	 *
 	 */
 
+	/**
+	 * Render file
+	 * 
+	 * You can either place custom files in /site/templates/AdminThemeUikit or you
+	 * can specify a custom folder via $config->AdminThemeUikitTemplates
+	 * 
+	 * Usage:
+	 * $adminTheme->render('_head.php');
+	 * @return string
```

Comment:
Since return type is documented in this docblock, shouldn't param $file be documented as well? Seems inconsistent :)

Reply:
you are absolutely right - that was a matter of lazyness :D